backend/src/test/java/com/cruru/applicant/service/ApplicantCardSorterTest.java

```diff
@@ -0,0 +1,86 @@
+package com.cruru.applicant.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.cruru.applicant.domain.dto.ApplicantCard;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("지원자 카드 sorter 테스트")
+class ApplicantCardSorterTest {
+
+    @DisplayName("지원자의 지원 날짜 기준으로 내림차순으로 정렬한다.")
+    @Test
+    void getCreatedAtComparator_desc() {
+        // given
+        ApplicantCard applicantCard1 = new ApplicantCard(1L, "최냥인",
+                LocalDateTime.of(2024, 10, 2, 20, 0), false, 5, 3.00, 1L);
+        ApplicantCard applicantCard2 = new ApplicantCard(2L, "최냥인",
+                LocalDateTime.of(2024, 10, 3, 20, 0), false, 0, 0.00, 1L);
+        List<ApplicantCard> applicantCards = Arrays.asList(applicantCard1, applicantCard2);
```

Comment:
```suggestion
        List<ApplicantCard> applicantCards = List.of(applicantCard1, applicantCard2);
```
들어가는 인자가 array라고 착각할 수도 있을 것 같아요~

Reply:
정렬을 해야 해서 List.of()는 불가능입니다..ㅎ ㅎ